provider/datagen/src/transform/cldr/time_zones/names.rs

```diff
@@ -105,3 +97,27 @@ impl IterableDataProvider<Bcp47ToIanaMapV1Marker> for crate::DatagenProvider {
         Ok(vec![Default::default()])
     }
 }
+
+fn compute_bcp47_ids_hash(bcp47_ids: &ZeroSlice<TimeZoneBcp47Id>) -> u64 {
+    let mut hasher = twox_hash::XxHash64::with_seed(0);
+    hasher.write(bcp47_ids.as_bytes());
+    hasher.finish()
+}
+
+#[test]
+fn test_compute_bcp47_ids_hash() {
+    let bcp47_ids: ZeroVec<TimeZoneBcp47Id> = [
+        TimeZoneBcp47Id(tinystr::tinystr!(8, "aedxb")),
+        TimeZoneBcp47Id(tinystr::tinystr!(8, "brfor")),
+        TimeZoneBcp47Id(tinystr::tinystr!(8, "usinvev")),
+    ]
+    .into_iter()
+    .collect();
+    let checksum1 = compute_bcp47_ids_hash(&bcp47_ids);
+    let mut hasher = twox_hash::XxHash64::with_seed(0);
+    for bcp47 in bcp47_ids.iter() {
+        hasher.write(bcp47.0.all_bytes());
+    }
+    let checksum2 = hasher.finish();
```

Comment:
uhm this basically tests that the implementation of `as_bytes` is the same as `all_bytes` for each entry. While this is useful to test, it should be a test in zeroslice, not here.

This should test things like: 
* If the order changes the checksums are different
* The checksum for `"abc", "def"` is different from the one of `"abcd", "ef"`
* The checksum for `"abc", "def"`, "" is different from the one for `"abc", "def", ""`
* The checksum for the hardcoded list is equal to a hardcoded checksum (stability)

Reply:
done